app/submitter/converter.py

```diff
@@ -92,11 +92,14 @@ def convert_answers_to_data(answer_store, questionnaire_json, routing_path):
             try:
                 answer_schema = answer_schema_list[answer['answer_id']]
             except KeyError:
-                answer_schema = None
+                logger.info("There is no answer id in the store", answer_id=answer['answer_id'])
```

Comment:
Change to `logger.error("No matching answer schema found in the store")` to avoid a second `KeyError`.